packages/linter/package.json

```diff
@@ -15,52 +15,31 @@
     "debug": "4.1.1",
     "execa": "2.0.5",
     "node-fetch": "2.6.0",
-    "probe-image-size": "5.0.0"
+    "prettier": "1.18.2",
+    "probe-image-size": "5.0.0",
+    "tap-parser": "10.0.0",
+    "typescript": "3.6.3"
```

Comment:
These can actually be removed if the scripts are invoked via e.g. `lerna run --scope '*/toolbox-linter' lint` from the root `amp-toolbox` directory instead of `npm run lint`. (Since lerna injects the appropriate binaries into the path.)

But developers probably expect to be able to run `npm x` here (after doing an `npm install`) so I think it's okay to duplicate these binaries.